src/components/SelectionListWithSections/MultiSelectListItem.tsx

```diff
@@ -53,7 +55,7 @@ function MultiSelectListItem<TItem extends ListItem>({
             alternateTextNumberOfLines={alternateTextNumberOfLines}
             onFocus={onFocus}
             shouldSyncFocus={shouldSyncFocus}
-            wrapperStyle={[wrapperStyle, styles.optionRowCompact]}
+            wrapperStyle={dynamicWrapperStyle}
```

Comment:
Let's make code simple.
```suggestion
            wrapperStyle={[wrapperStyle, styles.optionRowCompact, isAlternateTextMultilineSupported && {height: undefined}]}
```
And undo defining dynamicWrapperStyle